ports/winit/Cargo.toml

```diff
@@ -30,12 +30,13 @@ ProductName = "Servo"
 
 [features]
 debugmozjs = ["libservo/debugmozjs"]
-default = ["webdriver", "max_log_level"]
+default = ["webdriver", "max_log_level", "minibrowser"]
```

Comment:
I wonder if we could do away with this feature entirely. I think that the winit port doesn't use conditional compilation, so removing default features would break the build.

Reply:
done in 00d61872126774efdf70f91a26074ae1332c1a2e

Reply:
done in 00d61872126774efdf70f91a26074ae1332c1a2e